src/escalus_ct.erl

```diff
@@ -72,7 +72,16 @@ interpret_config_file_path(RelPath) ->
 rpc_call(Node, Module, Function, Args, TimeOut, Cookie) ->
     case is_ct_available() of
         true ->
-            ct_rpc:call(Node, Module, Function, Args, TimeOut, Cookie);
+            Result = ct_rpc:call(Node, Module, Function, Args, TimeOut, Cookie),
+            case Result of
+                {badrpc, Reason} ->
+                    ct:log("issue=rpc_call_failed "
+                            "node=~p function=~p:~p reason=~p",
```

Comment:
If 'ct:pal' is used here, the log will be put both to the file and console. I think this could be helpful.